package.json

```diff
@@ -41,20 +44,23 @@
     "./build/lib/util/fs.js": "./build/lib/util/fs-browser.js"
   },
   "dependencies": {
-    "babel-runtime": "^6.9.2",
+    "@babel/runtime": "^7.4.3",
```

Comment:
Could probably use a few version bumps here and there, since this PR has been open for a while. If you do go for Babel 7.5.x, do bear https://github.com/babel/babel/issues/10261 in mind.